hivemq4/base-image/docker-entrypoint.sh

```diff
@@ -2,83 +2,22 @@
 
 set -eo pipefail
 
-# Decode license and put file if present
-if [[ -n "${HIVEMQ_LICENSE}" ]]; then
-    echo "Decoding license..."
-    echo ${HIVEMQ_LICENSE} | base64 -d > /opt/hivemq/license/license.lic
-fi
-
-# We set the bind address here to ensure HiveMQ uses the correct interface. Defaults to using the container hostname (which should be hardcoded in /etc/hosts)
-if [[ -z "${HIVEMQ_BIND_ADDRESS}" ]]; then
-    echo "Getting bind address from container hostname"
-    ADDR=$(getent hosts ${HOSTNAME} | grep -v 127.0.0.1 | awk '{ print $1 }' | head -n 1)
+if [[ "${HIVEMQ_VERBOSE_ENTRYPOINT}" == "true" ]]; then
+    exec 3>&1
```

Comment:
In the k8s image, we are setting
```
if [[ "${HIVEMQ_VERBOSE_ENTRYPOINT}" == "true" ]]; then
  set -o xtrace
fi
```

Would it make sense to set the same here?

Reply:
was thinking about this too, adding it now; it's probably very convenient when debugging scripts that are sourced